CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ cmake_minimum_required(VERSION 3.5.1 FATAL_ERROR)
 #============================================================================
 # Initialize the project
 #============================================================================
-project(ignition-physics5 VERSION 5.2.0)
+project(ignition-physics5 VERSION 5.3.0)
```

Comment:
Can we just do a patch release since we haven't added any new functionality?

Reply:
I bumped the minor because of https://github.com/gazebosim/gz-physics/pull/390, which increases our required gz-cmake version to 2.13. If doesn't warrant a minor version bump; I'll revert to a patch bump

Reply:
I see. Let's go with minor bump then.